### PR TITLE
Fix join and remove command permission mistake and update doc&translate

### PR DESCRIPTION
**Pull request by lsCoding666**
## Description
Fix join and remove command permission mistake and update doc&translate
Show usage while command has not args.

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [x] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [x] Documentation

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My pull request is unique and no other pull requests have been opened for these changes
- [ ] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [ ] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/java/emu/grasscutter/command/commands/JoinCommand.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import static emu.grasscutter.utils.Language.translate;
 
 @Command(label = "join", usage = "join [AvatarIDs] such as\"join 10000038 10000039\"",
-        description = "commands.join.description", permission = "player.join")
+        description = "commands.join.description", permission = "server.join")
```
Comment (ghost) [inline src/main/java/emu/grasscutter/command/commands/JoinCommand.java]: I forgot to ask this, but can you also add a `permissionTargeted` so other players without the right permission cant force join/remove characters you dont want into your party?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/emu/grasscutter/command/commands/JoinCommand.java (hunk 1 above)
Comment (lsCoding666) [reply src/main/java/emu/grasscutter/command/commands/JoinCommand.java]: OK. I will hand on it.

Comment (lsCoding666) [reply src/main/java/emu/grasscutter/command/commands/JoinCommand.java]: > I forgot to ask this, but can you also add a `permissionTargeted` so other players without the right permission cant force join/remove characters you dont want into your party?

fixed

Comment (lsCoding666) [reply src/main/java/emu/grasscutter/command/commands/JoinCommand.java]: By the way, Is there any division standard between server permission and player permission? How can I choice?
player permission———client only?
server permission———both side?
But why
image
image

Comment (lsCoding666) [reply src/main/java/emu/grasscutter/command/commands/JoinCommand.java]: telepory can be excute on the both side.
image

**Comment by ghost** [comment]
The join/remove commands have been merged into the team command, please update your branch accordingly.

**Pull request closed without merging**